lib/_stream_duplex.js

```diff
@@ -52,10 +52,13 @@ var Writable = require('./_stream_writable');
 
 util.inherits(Duplex, Readable);
 
-var keys = objectKeys(Writable.prototype);
-for (var v = 0; v < keys.length; v++) {
-  var method = keys[v];
-  if (!Duplex.prototype[method]) Duplex.prototype[method] = Writable.prototype[method];
+{
+  // avoid scope creep, the keys array can then be collected
```

Comment:
this is now a very confusing comment as we rewrite consts/lets to var, but i guess there is nothing to do about that :joy:

Reply:
all this code is babelified anyway here. So we might even change some of this code in core.